daemon/containerd/image.go

```diff
@@ -44,21 +45,39 @@ func (i *ImageService) GetImage(ctx context.Context, refOrID string, options ima
 	}
 
 	cs := i.client.ContentStore()
-	conf, err := containerdimages.Config(ctx, cs, desc, platform)
-	if err != nil {
-		return nil, err
-	}
 
-	imageConfigBytes, err := content.ReadBlob(ctx, cs, conf)
+	var presentImages []ocispec.Image
+	err = i.walkImageManifests(ctx, desc, func(img *ImageManifest) error {
+		conf, err := img.Config(ctx)
+		if err != nil {
+			return err
+		}
+
+		imageConfigBytes, err := content.ReadBlob(ctx, cs, conf)
+		if err != nil {
+			return err
+		}
+
+		var ociimage ocispec.Image
+		if err := json.Unmarshal(imageConfigBytes, &ociimage); err != nil {
+			return err
+		}
+
```

Comment:
nit: we have a helper for that

```suggestion
		var ociimage ocispec.Image
		if err := readConfig(ctx, cs, conf, &ociimage); err != nil {
			return err
		}
```